src/components/EventItem/EventItem.tsx

```diff
@@ -0,0 +1,34 @@
+import React, {useContext} from 'react';
+import {Image, StyleSheet, Text, TouchableOpacity} from 'react-native';
+import {EventItemProps} from './EventItem.types';
+import {RootStoreContext} from '../../models';
+import {scale, verticalScale} from '../../utils';
+
+const EventItem = ({event}: EventItemProps) => {
+  const {events} = useContext(RootStoreContext);
+
+  return (
+    <TouchableOpacity onPress={() => events.getEventDetails(event.id, false)}>
+      <Image
+        source={{
+          uri:
+            event.image_url === ''
+              ? 'https://s3-media3.fl.yelpcdn.com/ephoto/B9oMfBlHwU_8oSe7blx_Lw/o.jpg'
```

Comment:
Why this hardcoded image url? Make image url mandatory. Also, In case we do need this can we keep this url in constants?

Reply:
Moved into constants.